hebiros_gazebo_plugin/include/hebiros_gazebo_pid.h

```diff
@@ -0,0 +1,25 @@
+#include "hebiros/PidGainsMsg.h"
+#include <cstdint>
+
+namespace hebiros {
+namespace sim {
+
+// A simple PID controller class
+class PidController {
+
+public:
+  PidController(double ff_scale)
+    : _ff_scale(ff_scale)
+  {}
+
+  double update(double target, double feedback, double dt, const PidGainsMsg& pid_gains, size_t gain_idx);
+
+private:
+  double _elapsed_error{};
+  double _prev_error{};
+  // TODO: scale here or when setting value?
+  double _ff_scale{};
```

Comment:
you're already initializing `ff_scale` in the constructor, so I'd probably just get rid of this default initialization